README.md

```diff
@@ -10,6 +10,13 @@ Expected environment:
   - Node Feature Discovery (NFD) Operator
   - Kernel Module Management (KMM) Operator ([Redhat docs](https://docs.openshift.com/container-platform/4.12/hardware_enablement/kmm-kernel-module-management.html), [codebase docs](https://kmm.sigs.k8s.io/documentation/deploy_kmod/))
 
+#### Build sources
+
+When building from source in a cluster the default path for the source location
+to download from github. This behaviour can be changed by editing [onload-sources.conf](base/onload-sources.conf).
+The format of the source must be in a `*.tar.gz` file which can be accessed and
+downloaded using an `ADD` command in a Dockerfile.
```

Comment:
Just making it clear when the sources are used and that we only currently only support building from source. Also that no modifications are required to the Dockerfiles.

```suggestion
When following the build steps below to build from source in a cluster, ensure the source location paths are correct for your environment. The default source location paths are
[onload](https://github.com/Xilinx-CNS/onload) and this repository. This behaviour can be changed by editing [onload-sources.conf](base/onload-sources.conf).
The format of the source files must be `*.tar.gz` as the following steps access and
download them using `ADD` commands in Dockerfiles.
```